src/Action.js

```diff
@@ -0,0 +1,28 @@
+export default class Action {
+	constructor (promise) {
+		this.promise = promise
+	}
+
+	// default onFulfilled action
+	/* istanbul ignore next */
+	fulfilled (p) {
+		this.promise._become(p)
+	}
+
+	// default onRejected action
+	rejected (p) {
+		this.promise._become(p)
+		return false
+	}
+
+	tryCall (f, x) {
```

Comment:
How about this: Add a comment to document why this method is structured the way it is, i.e. that we want a distinction between exceptions from internal `this.handle` and from user code.